content/induction/blocks/learning-as-community/index.md

```diff
@@ -0,0 +1,30 @@
++++
+title = 'Learning as a community 🏫'
+headless = true
+time="55"
+facilitation = true
+emoji= '🧩'
+[objectives]
+    1='Describe how we learn as a community at CYF'
+    2='Describe a typical week at Code Your Future'
++++
+
+At Code Your Future, we learn as a **community.**
+
+In this session, we're going to describe how we harness our community to overcome obstacles and develop our understanding.
+
+### Resources
+
+The facilitator will need to use this presentation 👉 [Learning as a community presentation](https://miro.com/app/board/uXjVPy92Cu8=/)
+
+### Preparation
+
+- [ ] Facilitator: Review this [presentation](https://miro.com/app/board/uXjVPy92Cu8=/) before class.
+- [ ] Facilitator: Ensure everyone can access the Miro board above.
+- [ ] Facilitator: Make sure everyone is split into groups of no more than 5.
+- [ ] Facilitator: Make sure every group has access to a laptop.
+- [ ] Facilitator: Make sure every group has access to a piece of paper and pen
+
+### Introduction
+
+The facilitator can use the Miro board presentation to guide trainees and volunteers on a discussion about how we learn together as a community.
```

Comment:
```suggestion
The facilitator will use the Miro board presentation to guide trainees and volunteers on a discussion about how we learn together as a community.
```

Reply:
I suggest you link them to a recording of you doing this if possible, to help the volunteer know what is expected.